Dockerfile.test

```diff
@@ -1,6 +1,6 @@
 FROM centos:8
 
-RUN yum module -y install go-toolset postgresql && yum -y install git
+RUN yum module -y install go-toolset postgresql && yum -y install git diffutils
```

Comment:
Where is it used? Isn't it because of database migrations? So it should be part of its PR.

Reply:
When golint fails, it tries to render output with diff.